assets/css/main.css

```
@@ -170,6 +170,28 @@ body {
   text-decoration: none;
 }
 
+#Explanation{
+  font-size: 12pt; 
+}
+
+#Toggle-Explanation{
+  background: #f1652991;
+  font-size: 12pt;
+  border: none;
+  color: white ;
+  padding: 12px 25px;
```

You can use `em` values here instead of `px`

```suggestion
  padding: .75em 1.5em;
```